setup.scripts/apache/mailwatch-apache.inc

```diff
@@ -24,26 +24,34 @@ function install-webserver {
             apacheBin="apache2"
         fi
     fi
+    logprint "Apache binary is $apacheBin"
 
     if [ "$apacheBin" == "httpd" ]; then
-        mwApacheConf="/etc/httpd/conf.d/mailwatch.conf"
+        mwApacheConf="/etc/httpd/conf.d/ssl_mailwatch.conf"
     elif [ "$apacheBin" == "apache2" ]; then
-        mwApacheConf="/etc/apache2/sites-available/mailwatch.conf"
+        mwApacheConf="/etc/apache2/sites-available/ssl_mailwatch.conf"
     fi
 
+    logprint "Using apache conf file $mwApacheConf"
+
     #backup old config
     if [ -f "$mwApacheConf" ]; then
         mv "$mwApacheConf" "$mwApacheConf.old"
     fi
 
 #    cp "$InstallFilesFolder/setup.scripts/apache/etc/apache2/conf-available/mailwatch.conf" /etc/apache2/conf-available/mailwatch.conf
-    cp "$InstallFilesFolder/setup.scripts/apache/etc/apache2/sites-available/mailwatch.conf" "$mwApacheConf"
+    cp "$InstallFilesFolder/setup.scripts/apache/etc/apache2/sites-available/ssl_mailwatch.conf" "$mwApacheConf"
```

Comment:
If we follow the "Debian rule", `ssl_mailwatch.conf` will be better as` mailwatch-ssl.conf`.

Reply:
At least for Centos it has to load after the ssl module (/etc/httpd/conf.d/ssl.conf)
But I guess we can keep the mailwatch-ssl.conf for debian https://github.com/mailwatch/mailwatch-install-script/pull/38/files/cbfef4fd0974269df318e3fe936e96df4949db4d..7b5765ec584c048801fafb3b0a68d40f0d9f9f13#diff-196ada4b2f7f4a03baef817c4fada680R32

Reply:
done